osbenchmark/builder/downloaders/builders/source_binary_builder.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import os
+
+from osbenchmark.builder.downloaders.builders.binary_builder import BinaryBuilder
+from osbenchmark.exceptions import ExecutorError, BuildError
+
+
+class SourceBinaryBuilder(BinaryBuilder):
+    def __init__(self, executor, path_manager, jdk_resolver, source_directory, build_jdk, log_directory):
```

Comment:
I'd rename it as ``build_jdk_version``, but up to you. The naming ``build_jdk`` mostly makes sense, it's about the jdk version, but there's a slight chance that people may think it's a boolean first then figure out "build a JDK" wound not make sense here.

Reply:
That's a good callout, will rename